test/unit/DCAHubCompanion/dca-hub-companion-multi-call-handler.spec.ts

```diff
@@ -104,30 +104,69 @@ contract('DCAHubCompanionMulticallHandler', () => {
     const AMOUNT_OF_SWAPS = 40;
     const SWAP_INTERVAL = 10000;
     const OWNER = '0x0000000000000000000000000000000000000003';
+    const MISC = ethers.utils.randomBytes(10);
```

Comment:
Beware, this random in tests might affect in a small way gas coverage (not really too important)

Reply:
Good point!